docs/developer-guide/etendo-copilot/how-to-create-copilot-tools.md

```diff
@@ -58,41 +58,28 @@ For this example, we will create a tool that will allow us to make a ping to a h
     *tools_deps.toml*: Contains the dependencies of the tools of the module. This file contains the dependencies of the tools of the module.
 
 
-2. Create the file `PingTool.py` in the `tools` folder. This file will contain the code of the tool. The code of the tool will be as follows, the comments explain the code:
-
-    ```python title="PingTool.py"
-    import os
-    from copilot.core.tool_wrapper import ToolWrapper # Import the ToolWrapper class from the copilot.core.tool_wrapper module. This class is the one that must be extended to create a new tool.
-
-    class PingTool (ToolWrapper):
-        name = 'PingTool' # Name of the tool
-        description = ('''This tool receives a hostname and returns the ping result.''') # Description of the tool. This description tells Copilot what the tool does and based on this description it will decide if this tool will solve the user's request.
-
-        def run(self, host: str, *args, **kwargs): # The run method is the one that will be executed when the tool is executed. The inputs of the method are the inputs of the tool.
-            import requests # Import the necessary libraries to execute the tool. It is recommended to import the libraries inside the run method to avoid conflicts with other tools.
-            response = requests.get(host)
-            return {"status_code": response.status_code}  # The run method must return a dictionary with the outputs of the tool.
-    ``` 
-    !!!Note 
-        The name of the tool must be the same as the name of the class that extends the ToolWrapper class.
+    
 
 3. In case the tool needs more than one input, its necessary to use a Dict as input. In order to do that, we have to create a new class that defines the inputs of the tool using pydantic. Here is an example of a tool that receives a Dict as input, with the structure of the Dict defined in a class:
+    
+    !!!Warning 
+        The name of the tool must be the same as the name of the class that extends the ToolWrapper class.
```

Comment:
Tool Search Key, is the one that has to match, the name can be different

<img width="387" alt="image" src="https://github.com/user-attachments/assets/06cc0a43-16a4-47ea-a211-249d26bf5022">.